WcaOnRails/app/models/user.rb

```diff
@@ -563,7 +565,11 @@ def admin?
   end
 
   def any_kind_of_delegate?
-    delegate_status.present?
+    !delegate_status.nil?
```

Comment:
Any specific reason why you changed this line?
I believe `present?` does exactly what you wrote doesn't it?

Reply:
Initially, I implemented in a way where `any_kind_of_delegate` will return `false` for trainee delegates, so I changed it to something like `!delegate_status.nil? && delegates_status != 'trainee_delegate'`, but later when I changed it, I just removed the second part. Yes, the old part was doing the same job, I'll change it back 😁